lib/timelineRow.d.ts

```diff
@@ -1,6 +1,7 @@
-import { TimelineKeyframe } from './timelineKeyframe';
-import { TimelineRowStyle } from './settings/styles/timelineRowStyle';
-import { TimelineRanged } from './timelineRanged';
-export interface TimelineRow extends TimelineRowStyle, TimelineRanged {
-    keyframes?: Array<TimelineKeyframe>;
-}
+import { TimelineKeyframe } from './timelineKeyframe';
+import { TimelineRowStyle } from './settings/styles/timelineRowStyle';
+import { TimelineRanged } from './timelineRanged';
+export interface TimelineRow extends TimelineRowStyle, TimelineRanged {
+    keyframes?: Array<TimelineKeyframe>;
+    title?: string;
```

Comment:
The title exists in the logic and in the demo, so I figured make one less ts ignore